tests/sqlparser_sqlite.rs

```diff
@@ -281,6 +281,104 @@ fn parse_create_table_gencol() {
     sqlite_and_generic().verified_stmt("CREATE TABLE t1 (a INT, b INT AS (a * 2) STORED)");
 }
 
+#[test]
+fn parse_create_table_on_conflict_col() {
+    let sql_rollback = "CREATE TABLE t1 (a INT, b INT ON CONFLICT ROLLBACK)";
+    match sqlite().verified_stmt(sql_rollback) {
```

Comment:
Will it be better to change it into a for-loop to simplify the test case.

```rust
 for keyword in ["ROLLBACK", "ABORT", "FAIL", "IGNORE", "REPLACE"] {
    let sql = "CREATE TABLE t1 (a INT, b INT ON CONFLICT {keyword})"
    ...
 }
```

Reply:
Completely agree, I modified the code accordingly